layout/header.php

```diff
@@ -15,126 +15,72 @@
 }
 asort($langsAssoc);
 ?>
-<!-- Topbar -->
-<nav class="sb-topnav navbar navbar-expand navbar-light bg-white shadow">
-    <!-- Navbar Brand -->
-    <a class="navbar-brand ps-3" href="/">Navbar</a>
 
-    <!-- Sidebar Toggle (Topbar) -->
-    <button class="btn btn-link btn-sm order-1 order-lg-0 me-4 me-lg-0" id="sidebarToggle"><i class="fas fa-bars"></i></button>
+<!-- Navbar -->
+<nav class="navbar fixed top-0 start-0 z-10 w-full bg-base-200 border-b border-gray-400">
+    <div class="navbar-start">
+        <a class="text-xl font-bold text-primary" href="/">Litcal.org</a>
```

Comment:
Here I would put just `LitCal`, not the domain, seeing the situation with the domain isn't quite clear. I'm still just using https://litcal.johnromanodorazio.com and https://litcal-staging.johnromanodorazio.com for the main and dev branches, until there's a clearer situation with the litcal.org domain... I have also created a logo we could put alongside here, I'll check it into the repo